f/missionConditions/fn_ColdBreath.sqf

```diff
@@ -1,19 +1,31 @@
 // F3 - Cold Breath
 // Credits and documentation: https://github.com/folkarps/F3/wiki
 // ====================================================================================
+f_var_coldBreathLoop = true;
```

Comment:
Is this variable ever set elsewhere? Or is it there for debug/admin purposes?

Reply:
It's not used in FA3 itself, but I wanted to have a way to turn it off, for example if the missionmaker wanted the weather to change and get hotter. It doesn't really have any overhead.

Reply:
That makes sense, was mainly checking that something wasn't missing